content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -37,208 +34,375 @@ localization:
+  de Venta` is selected.
+- :guilabel:`Invoice Line`: Specify the products with the correct taxes.
 
-- **Factura Electronica**: This is the regular document type applicable for Invoices, Credit Notes
-  and Debit Notes.
-- **Factura de Importación**: This should be selected for importation transactions.
-- **Factura de contingencia**: This is an exceptional type that is used as a manual backup if the
-  company is not able to use the ERP and if it is necessary to generate the invoice manually when
-  this invoice is added to the ERP.
+When done, click :guilabel:`Confirm`.
+
+.. image:: colombia/electronic-invoice-creation.png
+   :alt: Configuring the electronic invoice in Odoo.
 
 .. _colombia/invoice-validation:
 
 Invoice validation
-------------------
+~~~~~~~~~~~~~~~~~~
 
-After the invoice is validated, an XML file is created and sent automatically to Carvajal. This file
-is also displayed in the chatter.
+After the invoice confirmation, an XML file is created and sent automatically to Carvajal. The
+invoice is then processed asynchronously by the E-invoicing service UBL 2.1 (Colombia). The file is
+also displayed in the chatter.
 
-.. image:: colombia/carvajal-invoice-xml-chatter.png
+.. image:: colombia/invoice-sent.png
    :alt: Carvajal XML invoice file in Odoo chatter.
 
-The :guilabel:`Electronic Invoice Name` field is now displayed in the :guilabel:`Other Info` tab
+The :guilabel:`Electronic Invoice Name` field is now displayed in the :guilabel:`EDI Documents` tab
 with the name of the XML file. Additionally, the :guilabel:`Electronic Invoice Status` field is
 displayed with the initial value :guilabel:`In progress`.
 
+.. image:: colombia/invoice-sent-status.png
+   :alt: Electronic Invoice EDI Document XML file in the EDI Documents tab.
+
 .. _colombia/invoice-xml:
 
 Reception of legal XML and PDF
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The electronic invoice vendor receives the XML file and proceeds to validate the structure and the
-information in it. In the :guilabel:`Action` drop-down menu, select the :guilabel:`Check Carvajal
-Status` button. If everything is correct, the :guilabel:`Electronic Invoice Status` field value
-changes to :guilabel:`Validated`. Then, proceed to generate a legal XML, which includes a digital
-signature and a unique code (CUFE), a PDF invoice that includes a QR code, and the CUFE is also
-generated.
+The electronic invoice vendor (Carvajal) receives the XML file and proceeds to validate its
+structure and information.
 
-After this:
+In the :guilabel:`Action` menu, select the :guilabel:`Check Carvajal Status` button. The
+:guilabel:`Electronic Invoice Status` field value changes to :guilabel:`Validated` if everything
+is correct. Then, proceed to generate a legal XML, which includes a digital signature and a unique
+code (CUFE), a PDF invoice that consists of a QR code, and the previously-generated CUFE code.
```

Comment:
```suggestion
is correct. Then, proceed to generate a legal XML, which includes a digital signature and a unique
code (CUFE), a PDF invoice that consists of a QR code, and the previously-generated CUFE code.
```
The CUFE section is unclear, is a *2nd* CUFE code generated with the PDF? Or does it use the previously-generated CUFE code and includes that same code with the PDF? I changed the line, but please correct it if necessary